odoo/addons/base/views/res_partner_bank_views.xml

```diff
@@ -0,0 +1,132 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <record id="view_partner_bank_form" model="ir.ui.view">
+            <field name="name">res.partner.bank.form</field>
+            <field name="model">res.partner.bank</field>
+            <field name="priority">15</field>
+            <field name="arch" type="xml">
+                <form string="Bank account" name="bank_account_form" create="0">
+                    <sheet>
+                        <widget name="web_ribbon" title="Archived" bg_color="text-bg-danger" invisible="active"/>
+                        <group>
+                            <group>
+                                <field name="partner_id" invisible="1"/> <!-- Needed for default partner to be set for newly created records -->
+                                <field name="acc_number"/>
+                                <div class="d-flex" colspan="1" invisible="not show_clearing_number">
+                                    <field name="clearing_number_label" extra_labels="{
```

Comment:
This shows something is badly designed: just makes a point selection should not be used here.